cartography/intel/aws/rds.py

```diff
@@ -216,6 +216,114 @@ def load_rds_instances(
+            snapshot.vpc_id = rds_snapshot.VpcId,
+            snapshot.instance_create_time = rds_snapshot.InstanceCreateTime,
+            snapshot.master_username = rds_snapshot.MasterUsername,
+            snapshot.engine_version = rds_snapshot.EngineVersion,
+            snapshot.license_model = rds_snapshot.LicenseModel,
+            snapshot.snapshot_type = rds_snapshot.SnapshotType,
+            snapshot.iops = rds_snapshot.Iops,
+            snapshot.option_group_name = rds_snapshot.OptionGroupName,
+            snapshot.percent_progress = rds_snapshot.PercentProgress,
+            snapshot.source_region = rds_snapshot.SourceRegion,
+            snapshot.source_db_snapshot_identifier = rds_snapshot.SourceDBSnapshotIdentifier,
+            snapshot.storage_type = rds_snapshot.StorageType,
+            snapshot.tde_credential_arn = rds_snapshot.TdeCredentialArn,
+            snapshot.encrypted = rds_snapshot.Encrypted,
+            snapshot.kms_key_id = rds_snapshot.KmsKeyId,
+            snapshot.timezone = rds_snapshot.Timezone,
+            snapshot.iam_database_authentication_enabled = rds_snapshot.IAMDatabaseAuthenticationEnabled,
+            snapshot.processor_features = rds_snapshot.ProcessorFeatures,
+            snapshot.dbi_resource_id = rds_snapshot.DbiResourceId,
+            snapshot.original_snapshot_create_time = rds_snapshot.OriginalSnapshotCreateTime,
+            snapshot.snapshot_database_time = rds_snapshot.SnapshotDatabaseTime,
+            snapshot.snapshot_target = rds_snapshot.SnapshotTarget,
+            snapshot.storage_throughput = rds_snapshot.StorageThroughput,
+            snapshot.region = $Region,
+            snapshot.lastupdated = $aws_update_tag
+        WITH snapshot
+        MATCH (aa:AWSAccount{id: $AWS_ACCOUNT_ID})
+        MERGE (aa)-[r:RESOURCE]->(snapshot)
+        ON CREATE SET r.firstseen = timestamp()
+        SET r.lastupdated = $aws_update_tag
+    """
+    snapshots = []
+
+    for snapshot in data:
+        snapshots.append(snapshot)
+
+        snapshot['SnapshotCreateTime'] = dict_value_to_str(snapshot, 'EarliestRestorableTime')
+        snapshot['InstanceCreateTime'] = dict_value_to_str(snapshot, 'InstanceCreateTime')
+        snapshot['ProcessorFeatures'] = dict_value_to_str(snapshot, 'ProcessorFeatures')
+        snapshot['OriginalSnapshotCreateTime'] = dict_value_to_str(snapshot, 'OriginalSnapshotCreateTime')
+        snapshot['SnapshotDatabaseTime'] = dict_value_to_str(snapshot, 'SnapshotDatabaseTime')
+
+    neo4j_session.run(
+        ingest_rds_snapshot,
+        Snapshots=data,
+        Region=region,
+        AWS_ACCOUNT_ID=current_aws_account_id,
+        aws_update_tag=aws_update_tag,
+    )
+    _attach_snapshots(neo4j_session, snapshots, aws_update_tag)
+
+
+@timeit
+def _attach_snapshots(neo4j_session: neo4j.Session, snapshots: List[Dict], aws_update_tag: int) -> None:
+    """
+    Attach snapshots to their source instance
+    """
+    attach_member_to_source = """
```

Comment:
This query, along with `ingest_rds_snapshot` will be a great candidate for the new functionality on https://github.com/lyft/cartography/pull/1038, when it's soon ready.

Reply:
sounds good